update.xml

```diff
@@ -746,4 +746,17 @@
             <telegram>https://t.me/rasenkai</telegram>
         </z2_plus>
     </manufacturer>
+	<manufacturer id="Smartron">
```

Comment:
remove indentation a bit here

Reply:
removed

Reply:
no it's not
remove indentiation means to lower the spaces 
if you see starting of manufacturer tag is not aligned with ending

Reply:
now its removed